docs/source/operations/baf_verify.md

```diff
@@ -238,6 +238,41 @@ For final checking of the validity of the indy network.
 
 Upon successful connection, should display a `Pool Connected Successfully` msg.
 
+
+---  
+
+
+## R3 Corda Checks
+
+The flow chart shows the R3 Corda process. To verify the steps of deployment, follow the Verification Table 'R', to troubleshoot the general errors.
+
+![](./../_static/corda_flowchart.png)
+
+----
+### R3 Corda Troubleshooting
+### Table 'N'
+
+|Section|Sub-Section|Problem|Possible Cause|Solution|
+|----|---|----------|-------|--------------|--------|

```

Comment:
Table formatting needs to be corrected

Reply:
That is odd, the VS Code preview had no issues, let me check.

Reply:
Fixed